Source/Mapping.AutoMapper/TwoWayMappingProfile.cs

```diff
@@ -28,6 +28,23 @@ public virtual TSource Map(TDestination source)
             {
                 return mapper.Map<TDestination, TSource>(source);
             }
+
+            public virtual TSource Map(TDestination source, params (string parameterName, object parameterValue)[] parameters)
```

Comment:
Can we call the Map method on the base class to reduce code duplication? Or move the method contents to a protected static methods and call it from both Map methods?

Reply:
What I actually think we should do, is to deprecate the TwoWayMapper altogether. It's part of the old implementation and not really used in any of our newer projects anymore. We could think about adding "Obsolete" attribute to all constructs related to two-way mappng.

Reply:
I have removed the code duplication. 
I agree about TwoWayMapper. It usage really reduce code readability. I will add obsolete attribute.

Reply:
Should we then even add this new method to the two-way mapper? Probably not if we are going to remove those classes.

Reply:
Ok. Removed